### PR TITLE
Upgrade to Next.js 11

### PR DESCRIPTION
**Pull request by mewdriller**
- Leverage blur-up placeholder images
- Make images in FeaturedCaseStudy follow the mouse
- Fix some linting/conformance issues

### CODE REVIEW

**Comment by mewdriller** [comment]
I had an idea for some animation on the featured case study to have the images rotate in 3d to follow the mouse position. Along the way I ended up tweaking the mechanism for the layout in the grid of images. I _think_ the same technique would work for the team members layout?

File: src/components/HomePage/FeaturedCaseStudy.tsx [hunk 1]
```diff
@@ -242,47 +212,92 @@ export const FeaturedCaseStudy: FC = (props) => {
...
         <Gradient />
-        <ImagesContainer>
-          <ResponsiveImage
+        <motion.div
+          css={css`
+            --column-shift: ${spacing.xxl};
+
+            display: grid;
+            grid-template-columns: repeat(2, 1fr);
+            gap: ${spacing.xs};
```
Comment (chriswhawkins) [inline src/components/HomePage/FeaturedCaseStudy.tsx:232]: 🔤  alpha-ordering

File: src/components/HomePage/FeaturedCaseStudy.tsx [hunk 2]
```diff
@@ -242,47 +212,92 @@ export const FeaturedCaseStudy: FC = (props) => {
...
             width: calc(50% + ${GUTTER_SHIFT});
           }
         `}
       >
         <Gradient />
-        <ImagesContainer>
-          <ResponsiveImage
+        <motion.div
+          css={css`
+            --column-shift: ${spacing.xxl};
```
Comment (chriswhawkins) [inline src/components/HomePage/FeaturedCaseStudy.tsx:228]: ❤️

File: src/components/useMouseAnimationWhileVisible.ts [hunk 3]
```diff
@@ -51,12 +37,19 @@ export const useMouseAnimationWhileVisible = <T extends HTMLElement>(
           } = ref.current.getBoundingClientRect();
 
           // Transform the mouse position to a range of [-1, 1], [-1, 1] where 0,0 is the center of the reference element.
-          const elX = position.pageX - (left + window.pageXOffset);
-          const clampedX = Math.max(0, Math.min(elX, width));
-          const elY = position.pageY - (top + window.pageYOffset);
-          const clampedY = Math.max(0, Math.min(elY, height));
-          x.set(transform(clampedX, [0, width], [-1, 1]));
-          y.set(transform(clampedY, [0, height], [-1, 1]));
+          requestAnimationFrame(() => {
```
Comment (chriswhawkins) [inline src/components/useMouseAnimationWhileVisible.ts:40]: 👍

**Review by chriswhawkins — APPROVED** [review]
This looks great to me. I like the style/direction and how it's implemented here and can imagine building on as we add use-cases, etc. For mobile/touch devices, maybe a devicemotion event could replace mouse?

**Pull request merged**